src/cmake/openvino.cmake

```diff
@@ -100,13 +100,13 @@ install(TARGETS ${TARGET_NAME} EXPORT OpenVINOTargets
         RUNTIME DESTINATION ${OV_CPACK_RUNTIMEDIR} COMPONENT ${OV_CPACK_COMP_CORE} ${OV_CPACK_COMP_CORE_EXCLUDE_ALL}
         ARCHIVE DESTINATION ${OV_CPACK_ARCHIVEDIR} COMPONENT ${OV_CPACK_COMP_${archive_comp}} ${OV_CPACK_COMP_${archive_comp}_EXCLUDE_ALL}
         LIBRARY DESTINATION ${OV_CPACK_LIBRARYDIR} COMPONENT ${OV_CPACK_COMP_CORE} ${OV_CPACK_COMP_CORE_EXCLUDE_ALL}
-        NAMELINK_COMPONENT ${OV_CPACK_COMP_CORE_DEV}
+        # NAMELINK_COMPONENT ${OV_CPACK_COMP_CORE_DEV}
```

Comment:
TODO: core_dev target installs symbolic links and whee, need to exclude them from the wheel